userguide/object_detection/data-preparation.md

```diff
@@ -0,0 +1,134 @@
+# IG02 Data Preparation
+
+In this section, we will show you how to download a publicly available dataset
+and load it into an SFrame. This will allow you to try out the Object Detection
+toolkit for yourself. To get this dataset into the format expected by our
+toolkit, we will rely on many useful SFrame functions.
+
+*Note: Please make sure that you have Turi Create 4.3 or above for these steps*
+
+The dataset that we will use is [INRIA Annotations for Graz-02
+(IG02)](https://lear.inrialpes.fr/people/marszalek/data/ig02/). Our goal is to
+make a detector that can spot bikes and cars. Go ahead and click this link and
+download `ig02-v1.0-bikes.zip` and `ig02-v1.0-cars.zip`. In this tutorial, we
+will assume that you have unzipped these files into `~/Downloads/ig02`. That
+is, you should have the following data structure:
+
+```
+~/Downloads/ig02/
+    bike/
+        bike_001.image.png
+        bike_001.mask.0.png
+        bike_001.mask.1.png
+        bike_001.mask.2.png
+        bike_001.mask.3.png
+        bike_002.image.png
+        ...
+    cars/
+        carsgraz_001.image.png
+        carsgraz_001.mask.0.png
+        carsgraz_002.image.png
+        ...
+```
+
+This dataset contains masks marking the exact pixels belonging to each object
+instance. Each image may have several masks to indicate the presence of
+multiple objects. Red pixels indicate object presence and green indicate
+presence but with [occlusion](https://en.wikipedia.org/wiki/Occultation). For
+object detection, we will convert this richly annotated data to bounding boxes.
+Here is a snippet to turn this into an SFrame:
+
+```python
+import turicreate as tc
+import numpy as np
```

Comment:
Looks like you're not using `glob`. Also you're only using `numpy` from your helper function, so I would just move the import to your helper function.

Reply:
:+1: